src/tests/dictionaryConcepts/components/CreateMapping.test.js

```diff
@@ -74,4 +75,14 @@ describe('Test suite for dictionary concepts components', () => {
     const inputs = wrapper.find('select');
     expect(inputs).toHaveLength(2);
   });
+  it('should render the new relationship which is equal to Associated With', () => {
+    wrapper = mount(<CreateMapping {...props} />);
```

Comment:
Can you describe how this ticket is related to the ticket description?

Reply:
The test is checking that when that the source is changed the relationship can also be adjusted.  It is related to my ticket because it is what I was fixing. As per the ticket description. When a user changed the source from CIEL to anything else, the user could not adjust the relationship field.

Reply:
Can the description explicitly state what should lead to the rendering of the new relationship?

Reply:
I have updated the description. Is it explicit enough?